src/repository.js

```diff
@@ -11,3 +11,8 @@ exports.findPersona = async (id) => {
   const db = await client.getDB()
   return db.collection('persona').findOne({ _id: ObjectID(id) })
 }
+
+exports.updatePersona = async (id, data) => {
+  const db = await client.getDB()
+  return db.collection('persona').update({ _id: ObjectID(id) }, data, {upsert:false})
```

Comment:
Better use `updateOne`

Reply:
Also prefer `module.exports` over `exports`

https://blog.tableflip.io/the-difference-between-module-exports-and-exports/